types/leaflet-freedraw/index.d.ts

```diff
@@ -182,3 +182,27 @@ declare module "leaflet" {
         off(type: 'markers', fn?: MarkerEventHandler, context?: any): this;
     }
 }
+
+type localCreate = typeof CREATE;
+type localEdit = typeof EDIT;
+type localDelete = typeof DELETE;
+type localAppend = typeof APPEND;
+type localEditAppend = typeof EDIT_APPEND;
+type localNone = typeof NONE;
+type localAll = typeof ALL;
+
+declare namespace GlobalFreeDraw {
```

Comment:
```suggestion
declare namespace FreeDraw {
```
it's `FreeDraw.CREATE` in source code:
https://github.com/Wildhoney/Leaflet.FreeDraw/blob/master/src/FreeDraw.js#L345

Reply:
Shouldn’t I be worried about the case where window is undefined ? I mean it’s an extremely contrived example but :
I’m in react, I’m doing some server rendered app, where I’m writing a component.
I decide to `import FreeDraw from “leaflet-freedraw”` in my component and for some reason I’m using `FreeDraw.ALL` in my component, let’s say as a prop to another component which will not be ssr rendered.

because it’s running in ssr and thus have no window object. That piece of code in the if(window != undefined) is never ran, and FreeDraw and globalFreeDraw are not merged together.

I’m sorry I am a complete typescript beginner so I might be too paranoid in this case.

Reply:
Ok I just did a test, this package is completely node unfriendly as it use window in other places (requestAnimationFrame).

So I was paranoid over nothing, it's unusable in SSR. I'll fix these (also need to change the tests on top of your changes).